jobs.md

```diff
@@ -38,7 +38,7 @@ condor_status -autoformat Machine GalaxyGroup GalaxyDockerHack | grep hack | sor
 The following command is failing all jobs of the service-account user.
 
 ```bash
-gxadmin query queue-details | grep  service-account | awk '{print $3}' |  xargs -I {} sh -c "gxadmin mutate fail-job {}"
+gxadmin tsvquery jobs --user=service-account --nonterminal | awk '{print $1}' |  xargs -I {} -n 1 gxadmin mutate fail-job {} --commit 
```

Comment:
the n1 can be removed isn't it?

Reply:
it is redundant with `-I`, yes. I have always added it for explicitness, but, either is fine.